content/en/docs/instrumentation/net/exporters.md

```diff
@@ -173,7 +178,10 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     .Build();
 ```
 
-## Prometheus
+## Prometheus (Experimental)
+
+***Note:** this is experimental and dependent on the OpenTelemetry
+specification to be made stable before it will be a released package. For now, we recommend using the OTLP exporter and using the OpenTelemetry Collector to send metrics to Prometheus*
```

Comment:
I do not think we should make any recommendation about using this vs OTLP Exporter. (for eg: not all users have the infra to run collector). Simply state the package is non-stable pending spec.

Reply:
The docs are about helping people, therefore using that approach, which is well estabilished is perfect for this document.